Essentials/src/main/java/com/earth2me/essentials/utils/LocationUtil.java

```diff
@@ -267,7 +267,7 @@ public static boolean shouldFly(IEssentials ess, final Location loc) {
         int y = (int) Math.round(loc.getY());
         final int z = loc.getBlockZ();
         int count = 0;
-        while (LocationUtil.isBlockUnsafe(ess, world, x, y, z) && y > -1) {
+        while (LocationUtil.isBlockUnsafe(ess, world, x, y, z) && y > ess.getWorldInfoProvider().getMinHeight(world) - 1) {
```

Comment:
equal to or greater than moment?

Reply:
ok

Reply:
too many space changes, applied manually